oat-sa/extension-tao-proctoring
# always show "change time" button TCA-571

https://oat-sa.atlassian.net/browse/TCA-571

**Description:**
Proctor want to be able to adjust time and see which adjustments have been done so that he can have an overview of all performed time adjustments.

**How to test:**
- switch taoProctoring in your TAO instance to this branch
- create delivery with proctoring and open it as test taker
- approve it as proctor, be sure timer button visible but disabled
- pause a test as testtaker
- refresh sessions list as proctor and ensure that timer and authorize buttons are enabled
